Mathlib/LinearAlgebra/RootSystem/Defs.lean

```diff
@@ -27,10 +29,12 @@ This file contains basic definitions for root systems and root data.
 
 ## Todo
 
-* Introduce the Weyl Group
-* Base change of root pairings.
+* Put more API theorems in the Defs file.
```

Comment:
If this would introduce more imports, I suggest putting the API in `Lemmas` in order to keep `Defs` lightweight.